recipe/meta.yaml

```diff
@@ -202,6 +204,18 @@ outputs:
         # ensure only one test is left (namely, our hint)
         - python -c "import sys; lines=open('testlog').readlines(); sys.exit(0 if any('======== 1 failed' in x for x in lines) else 1)"
 
+  - name: scipy-typed
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('scipy', exact=True) }}
+        - scipy-stubs >={{ version }}.0,<{{ next_micro_version }}
+    test:
+      imports:
+        - scipy
```

Comment:
is there something typing related we can test here? `import scipy` obviously shouldn't break, but that's also unlikely from just adding other run-dependencies.

Reply:
I suppose `stubtest` would be most appropriate in this case. That requires `basedmypy` run (`mypy` would reuslt in false positives), which is conda-forge: https://anaconda.org/conda-forge/basedmypy

Reply:
@h-vetinari is there a good way to include the Mypy config files from scipy-stubs so that they can be used by `stubtest` in the tests here?

Reply:
In principle, `scipy-stubs` could ship the mypy files, but I have to wonder why we're adding this output to scipy at all. It seems like it would fit much better as an additional output to `scipy-stubs` (not just due to the version relationship, but also because then it could easily make use of the mypy config files).

Reply:
I think I agree actually, let me make a PR over there.